source/localizable/smart_cart/index.html.md.erb

```diff
@@ -0,0 +1,58 @@
+---
+parent: smart_cart
+title: overview
+---
+
+# Overview <%= edit_link %>
+
+There are currently two tools for Merchants that can be used to integrate Smart Cart orders
+into their existing system, the [Orders API](/smart_cart/orders_api) and
+[Webhook](/smart_cart/webhook).
+
+<%= partial 'partials/toc' %>
+
+## Orders API
+
+Orders API allows Skroutz Merchants to perform actions on their Smart Cart orders,
+such as accepting or rejecting an order.
+
+[Orders API documentation](/smart_cart/orders_api)
+
+## Webhook
+
+Webhook allows Skroutz Merchants to automatically receive order details to their
+platform when a new Smart Cart order is placed or an existing one is updated.
+
+[Webhook documentation](/smart_cart/webhook)
+
+## Use cases
+
+Both [Webhook](/smart_cart/webhook) and [Orders API](/smart_cart/orders_api) can be enabled and
+used separately. For a full orders integration though, it would require to implement both services,
+since Webhook can be utilized to get notified about new orders (or order updates) and the Orders
+API can be used to interact with that orders later on.
+
+An order coming in through a Webhook event has a unique `code`. You can use this code to
```

Comment:
I suggest "coming through" instead of "coming in through"

Reply:
OK ee8901b8